SimDataFormats/TrackingAnalysis/src/classes_def.xml

```diff
@@ -23,6 +23,8 @@
 
   <class name="TrackingParticleRefVector" />
   <class name="edm::Wrapper<TrackingParticleRefVector>"/>
+
+  <class name="std::map<edm::Ptr<TrackingParticle>,edm::RefToBase<TrackingParticle> >"/>
```

Comment:
I'm still curious why `RefToBase<TrackingParticle>` is be needed. I don't see anything inheriting from `TrackingParticle`, so a polymorphic reference is not really needed (i.e. `edm::Ref` would suffice), and for any new use for polymorphic Refs the `edm::Ptr` is strongly preferred.